prowler/providers/kubernetes/services/kubelet/kubelet_config_yaml_permissions/kubelet_config_yaml_permissions.py

```diff
@@ -0,0 +1,25 @@
+from prowler.lib.check.models import Check, Check_Report_Kubernetes
+from prowler.lib.utils.utils import get_file_permissions
+from prowler.providers.kubernetes.services.core.core_client import core_client
+
+
+class kubelet_config_yaml_permissions(Check):
+    def execute(self) -> Check_Report_Kubernetes:
+        findings = []
+        for node in core_client.nodes:
+            report = Check_Report_Kubernetes(self.metadata())
+            report.namespace = node.namespace
+            report.resource_name = node.name
+            report.resource_id = node.uid
+            # It can only be checked if Prowler is being executed inside a worker node
+            if node.inside:
+                report.status = "PASS"
+                report.status_extended = f"kubelet config.yaml file permissions are set to 600 or more restrictive in Node {node.name}."
+                if get_file_permissions("/var/lib/kubelet/config.yaml") > 0o600:
```

Comment:
This will raise an exception if you can't recover the permissions, thus the function returns `None`
```
>>> None > 0o600
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: '>' not supported between instances of 'NoneType' and 'int'
```